docs/api_documents/style1.css

```diff
@@ -0,0 +1,408 @@
+/* Asciidoctor default stylesheet | MIT License | http://asciidoctor.org */
+/* Remove comment around @import statement below when using as a custom stylesheet */
```

Comment:
Could we simply use an adoc file instead of 10+ files? What do we need html, css etc?

Reply:
Hello Liguang,

Absolutely, we will use only 1 adoc and maybe 0 or 1 css file eventually. Currently there are many files, but they are candidate files and I will remove once we choose one of them.

Reply:
Cool thanks.